source/includes/auto-encryption-options.rst

```diff
@@ -0,0 +1,128 @@
+The following table describes the structure of an
+``AutoEncryptionOptions`` object:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 30 70
+
+   * - Name
+     - Description
+
+   * - **keyVaultClient**
+     - **Type:** ``MongoClient``
+
+       **Description**
```

Comment:
Suggestion: Separate lines for Description (bolded) and the descriptive (plain) text.
\